src/components/ActionSheet/ActionSheetDropdown.tsx

```diff
@@ -9,14 +9,18 @@ import "./ActionSheet.css";
 
 const stopPropagation: React.MouseEventHandler = (e) => e.stopPropagation();
 
+export type ActionSheetDropdownProps = Omit<
+  SharedDropdownProps,
+  "popupDirection"
```

Comment:
```suggestion
  "popupDirection" | "popupOffsetDistance"
```

тогда и `popupOffsetDistance`